app/src/main/res/values/type.xml

```diff
@@ -5,6 +5,7 @@
     <item name="android:textSize">40sp</item>
```

Comment:
The commit message seems incomplete, please recheck?

Reply:
Actually, that is it 😅 I have added the reasoning in the description.

Reply:
I meant the description itself seems to be incomplete.